site/en/docs/extensions/whatsnew/index.md

```diff
@@ -16,57 +16,149 @@ updated: 2021-08-04
 Check this page often to learn about changes to the Chrome extensions platform, its documentation,
 and related policy or other changes.
 
-### Review violation troubleshooting updates
-
-#### Published 2021.10.27
+### Chrome 96: Dynamic content scripts {: #m96-dynamic-content-scripts }
+
+The [`chrome.scripting`](/docs/extensions/reference/scripting/) API now supports
+[registering](/docs/extensions/reference/scripting/#method-registerContentScripts),
+[updating](/docs/extensions/reference/scripting/#method-updateContentScripts),
+[unregistering](/docs/extensions/reference/scripting/#method-unregisterContentScripts), and [getting
+a list](/docs/extensions/reference/scripting/#method-getRegisteredContentScripts) of content scripts
+at runtime. Previously, content scripts could only be statically declared in an extension's
+manifest.json or programmatically injected at runtime with
+[`chrome.scripting.executeScript()`](/docs/extensions/reference/scripting/#method-executeScript).
+
+### Chrome 96: Massively expanded promise support {: #m96-promise-support }
```

Comment:
I meant to catch this before it was merged. In numbers what do 'massively' and 'significantly more' actually mean? Did you double the number of promises? Higher? 

The suggestion was described as 'fun'. The problem is that one bit of fun in an otherwise stoic information dump has a danger of being read a hyperbole. We need to avoid that.

cc: @rdcronin

Reply:
More than doubled, I think.

How about:

Expanded promise support to 27 more APIs

Reply:
Sounds good to me.
```suggestion
### Chrome 96: Expanded promise support to 27 more APIs {: #m96-promise-support }
```